addons/event_crm/models/event_registration.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    lead_ids = fields.Many2many('crm.lead', string='Leads', readonly=True, help="Leads generated from the registration.")
+
+    def _get_registration_lead_values(self, rule_type, registration_ids=False):
```

Comment:
business method should be declared at the end of the file. See guidelines -> https://www.odoo.com/documentation/master/reference/guidelines.html#symbols-and-conventions at 
**In a Model attribute order should be**